test-stats.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python3.6
```

Comment:
Why 3.6 ?? if container has 3.5 or 3.7 3.3 ???

Reply:
Updated in the way it was before